docs/vscode.md

```diff
@@ -3,50 +3,75 @@
-- [GitLens](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens)
-- [yarn](https://marketplace.visualstudio.com/items?itemName=gamunu.vscode-yarn)
-- [vscode-icons](https://marketplace.visualstudio.com/items?itemName=vscode-icons-team.vscode-icons)
-- [export-index](https://marketplace.visualstudio.com/items?itemName=BrunoLM.export-index)
+-   [Debugger for Firefox](https://marketplace.visualstudio.com/items?itemName=firefox-devtools.vscode-firefox-debug)
+-   [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome)
+-   [GitLens](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens)
+-   [yarn](https://marketplace.visualstudio.com/items?itemName=gamunu.vscode-yarn)
+-   [vscode-icons](https://marketplace.visualstudio.com/items?itemName=vscode-icons-team.vscode-icons)
+-   [export-index](https://marketplace.visualstudio.com/items?itemName=BrunoLM.export-index)
 
 The ESLint and Prettier integrations can be configured to fix and format your code on save.
 The Python integration can be configured to use Black as a formatter and run checks with Pylint,
 whether they are globally installed or in a virtualenv. Sample `.vscode/settings.json`:
 
 ```json
 {
-  "[typescript]": {
-    "editor.formatOnSave": true
-  },
-  "[typescriptreact]": {
-    "editor.formatOnSave": true
-  },
-  "[python]": {
-    "editor.formatOnSave": true
-  },
-  "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true,
-    "source.organizeImports": false
-  },
-  "python.testing.pytestArgs": [
-    "flask"
-  ],
-  "python.testing.unittestEnabled": false,
-  "python.testing.nosetestsEnabled": false,
-  "python.testing.pytestEnabled": true,
-  "python.formatting.provider": "black",
-  "python.formatting.blackPath": "flask/venv/bin/black",
-  "python.linting.pylintEnabled": true,
-  "python.linting.enabled": true
+    "[typescript]": {
+        "editor.formatOnSave": true
+    },
+    "[typescriptreact]": {
+        "editor.formatOnSave": true
+    },
+    "[python]": {
+        "editor.formatOnSave": true
+    },
+    "editor.codeActionsOnSave": {
+        "source.fixAll.eslint": true,
+        "source.organizeImports": false
+    },
+    "python.testing.pytestArgs": ["flask"],
+    "python.testing.unittestEnabled": false,
+    "python.testing.nosetestsEnabled": false,
+    "python.testing.pytestEnabled": true,
+    "python.formatting.provider": "black",
+    "python.formatting.blackPath": "flask/venv/bin/black",
+    "python.linting.pylintEnabled": true,
+    "python.linting.enabled": true
 }
 ```
+
+## Debugging in the Flask container
+
+-   [Debugging in VSCode](https://code.visualstudio.com/docs/editor/debugging)
+-   [Remote - Containers extension](https://code.visualstudio.com/docs/remote/containers)
+-   [Using profiles with Compose](https://docs.docker.com/compose/profiles/)
+
+To directly enter a Docker development that supports debugging in the container and launch the debugger:
+
```

Comment:
I think we can delete this line as well?

Reply:
This section presents a set of steps to bring up the debugger directly, whereas the next section presents how to bring up the debugger while there's a Flask app running. However, I will delete this section and only keep the second one.